kinesis-video-pic/src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -185,6 +185,7 @@ extern "C" {
 #define STATUS_MAX_FRAME_TIMESTAMP_DELTA_BETWEEN_TRACKS_EXCEEDED                    STATUS_CLIENT_BASE + 0x00000085
 #define STATUS_STREAM_SHUTTING_DOWN                                                 STATUS_CLIENT_BASE + 0x00000086
 #define STATUS_CLIENT_SHUTTING_DOWN                                                 STATUS_CLIENT_BASE + 0x00000087
+#define STATUS_PUTMEDIA_LAST_PERSIST_ACK_NOT_RECEIVED								STATUS_CLIENT_BASE + 0x00000088

```

Comment:
wondering how this  status could help the user in terms of :
a) there is merely a logging status stating that we could not get the ACK within the ACK wait interval time but connection has been reset ? or
b) can the user retry the last fragment by cancelling the reset?

Reply:
Right now this is serving as a logging status. There is not much user can do to this status because the buffer may already be gone.